prql-compiler/src/parser/mod.rs

```diff
@@ -824,6 +824,18 @@ Canada
         parse_expr("_2").unwrap().kind.into_ident().unwrap();
         parse_expr("_").unwrap().kind.into_ident().unwrap();
 
+        assert_yaml_snapshot!(parse_expr(r#"add 1. 2"#).unwrap(), @r###"
+        ---
+        Binary:
+          left:
+            Literal:
+              Integer: 2
+          op: Add
+          right:
+            Literal:
+              Integer: 2
+        "###);
```

Comment:
Shouldn't this be:

```yaml
FuncCall:
  name:
    Ident: add
  args:
  - Literal:
      Float: 1.0
  - Literal:
      Integer: 2
```

Reply:
Yes! This is the test that's currently failing. I should have left the result blank to make that clear...